modules/serverless-creating-broker-labelling.adoc

```diff
@@ -2,10 +2,10 @@
 //
 // * serverless/event_workflows/serverless-using-brokers.adoc
 
-[id="serverless-creating-broker-admin_{context}"]
-= Creating a broker by labeling a namespace
+[id="serverless-creating-broker-labelling_{context}"]
+= Creating a broker by labelling a namespace
 
-If you have cluster administrator permissions, you can create the `default` broker automatically by labeling a namespace.
+You can create the `default` broker automatically by labelling a namespace.
```

Comment:
I don't want to complicate things here, but a user can only add labels to the namespaces that they _own_ and have write permissions, too (even when they don't have to be cluster admins). But maybe this is already clear to the user  ? (one could add "by labelling a namespace to which you have write access." or so).

Reply:
Thanks for clarifying, I'll add that since I think it makes sense to clarify it.